src/app/store/index.ts

```diff
@@ -1,35 +1,38 @@
-import {
-  ActionReducer,
-  ActionReducerMap,
-  createFeatureSelector,
-  createSelector,
-  MetaReducer
-} from '@ngrx/store';
-import { environment } from '../../environments/environment';
-import * as fromCountries from './countries/countries.reducer';
-import * as fromCompany from '../modules/registration/store/company.reducer';
-import * as fromCart from './cart/cart.reducer'
-import * as fromSectors from './sectors/sectors.reducer';
-import * as fromLogin from './login/login.reducer';
+import {

```

Comment:
Something wrong this changes in this file. May be line endings? May you check please?